bindings/netstandard/ElectionGuard/ElectionGuard.Shared/ElGamal.cs

```diff
@@ -71,4 +147,20 @@ protected override unsafe void DisposeUnmanaged()
             Handle = null;
         }
     }
+    public static unsafe class Elgamal
+    {
+        unsafe public static ElGamalCiphertext Encrypt(
+            ulong plaintext, ElementModQ nonce, ElementModP publicKey)
+        {
+            var status = NativeInterface.ElGamal.Encrypt(
+                    plaintext, nonce.Handle, publicKey.Handle,
+                    out NativeElGamalCiphertext ciphertext);
+            if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
+            {
+                Console.WriteLine($"Encrypt Error Status: {status}");
```

Comment:
This seems repeated a bunch. Is there a way of building a method that'll just do 
```
log_error(status)

$"{name_of_method} Error Status: {status}
```
Maybe it could do the check as well? This would reduce the references to the enumeration and shift it to the method.

Reply:
#136